plug.vim

```diff
@@ -775,6 +776,18 @@ function! s:update_impl(pull, force, args) abort
     let s:py = s:version_requirement(
           \ map(split(split(pyv)[0], '\.'), 'str2nr(v:val)'), [2, 6])
   endif
+
+  " TODO: Remove when jobwait() blocks correctly
+  if has('nvim') && has('vim_starting')
+    if s:py
+      let s:nvim = 0
+    else
+      echohl WarningMsg
+      echomsg 'vim-plug: +PlugInstall/+PlugUpdate synchronous only with python support'
+      echomsg 'vim-plug: pip(3) install neovim'
+      echohl None
+    endif
+  endif
```

Comment:
@junegunn Right you are, will patch that once I've made the fix for the seg fault. I think it was just a matter of locking. Might have over zealously removed some.